specification/translation/Azure.AI.DocumentTranslation/models.tsp

```diff
@@ -470,3 +435,52 @@ model FileFormat {
   @doc("Supported Type for this format")
   type?: string;
 }
+
+@doc("Document Translate Request Body")
+model DocumentTranslateBody {
+  @doc("""
+    Specifies source language of the input document.
+    If this parameter isn't specified, automatic language detection is applied to determine the source language.
+    For example if the source document is written in English, then use sourceLanguage=en
+    """)
+  @query("sourceLanguage")
+  sourceLanguage?: string;
+
+  @doc("""
+    Specifies the language of the output document.
+    The target language must be one of the supported languages included in the translation scope.
+    For example if you want to translate the document in German language, then use targetLanguage=de 
+    """)
+  @query("targetLanguage")
+  targetLanguage: string;
+
+  @doc("""
+    A string specifying the category (domain) of the translation. This parameter is used to get translations 
+    from a customized system built with Custom Translator. Add the Category ID from your Custom Translator 
+    project details to this parameter to use your deployed customized system. Default value is: general.
+    """)
+  @query("category")
+  category?: string = "general";
+
+  @doc("""
+    Specifies that the service is allowed to fall back to a general system when a custom system doesn't exist. 
+    Possible values are: true (default) or false.
+    """)
+  @query("allowFallback")
+  allowFallback?: boolean;
+
+  @doc("Content Type as multipart/form-data")
+  @header
+  contentType: "multipart/form-data";
```

Comment:
Why do we move these, instead of keeping them in `DocumentTranslateContent`?

Reply:
There is a bug for current autorest emitter, see previous commit and previous comment.